app/src/core/design/components/userComponents/Video/Video.js

```diff
@@ -2,12 +2,23 @@ import { useNode } from "@craftjs/core";
 import { Grid } from "@material-ui/core";
 import React from "react";
 import { VideoDefaultProps, VideoSettings } from "./VideoSettings";
-export const Video = ({ props, style, parentStyle, ...rest }) => {
+import { CloudinaryContext, Transformation } from "cloudinary-react";
+import { Image } from "cloudinary-react";
+
+export const Video = ({ props, style, defaultThumbnail, parentStyle, ...rest }) => {
     const {
         connectors: { connect, drag },
         id
     } = useNode();
 
+    if (!String.prototype.format) {
```

Comment:
we can move this to utils maybe because it will be used by other components also